.taskcluster.yml

```diff
@@ -58,6 +58,39 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("grcov_test_task")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          env:
+            CODECOV_TOKEN: 1344bbfa-8d56-481e-9cb8-281ba0f9b6af
```

Comment:
Let's use the mozilla/rust-code-analysis token: `4df01912-087e-489a-be28-25aa911cb9d2`